lib/bootstrap/bundler.rb

```diff
@@ -130,11 +141,19 @@ def bundler_arguments(options = {})
       if options[:install]
         arguments << "install"
         arguments << "--clean" if options[:clean]
+        if options[:local]
+          arguments << "--local"
+          arguments << "--no-prune"
```

Comment:
Maybe add a comment to explain what `--no--prune` is actually doing,  I had to check the bundler doc and read:
`Don't remove stale gems from the cache.`